modules/core/src/test/java/org/apache/synapse/endpoints/dynamic/InboundEndpointTestCase.java

```diff
@@ -0,0 +1,111 @@
+            + "    <parameter name=\"inbound.http.port\">8085</parameter>\n" + "   </parameters>\n"
+            + "</inboundEndpoint>";
+    private static final String sampleEP2 = "<inboundEndpoint xmlns=\"http://ws.apache.org/ns/synapse\"\n"
+            + "                name=\"HttpListenerEP2\"\n" + "                sequence=\"TestIn\"\n"
+            + "                onError=\"fault\"\n" + "                protocol=\"http\"\n"
+            + "                suspend=\"false\">\n" + "   <parameters>\n"
+            + "    <parameter name=\"inbound.http.port\">8085</parameter>\n" + "   </parameters>\n"
+            + "</inboundEndpoint>";
+    private static final String sampleEP3 = "<inboundEndpoint xmlns=\"http://ws.apache.org/ns/synapse\"\n"
+            + "                name=\"HttpListenerEP3\"\n" + "                sequence=\"TestIn\"\n"
+            + "                onError=\"fault\"\n" + "                protocol=\"http\"\n"
+            + "                 statistics=\"enable\" trace=\"enable\">\n" + "   <parameters>\n"
+            + "    <parameter name=\"inbound.http.port\">8085</parameter>\n" + "   </parameters>\n"
+            + "</inboundEndpoint>";
+    private static final String sampleEP4 =
+            "<inboundEndpoint xmlns=\"http://ws.apache.org/ns/synapse\"\n" + "name=\"\"\n"
+                    + "                sequence=\"TestIn\"\n" + "onError=\"fault\"\n"
+                    + "                protocol=\"http\"\n"
+                    + "                 statistics=\"enable\" trace=\"enable\">\n" + "<parameters>\n"
+                    + "  <parameter name=\"inbound.http.port\">8085</parameter>   </parameters>\n"
+                    + "</inboundEndpoint>";
+    private static final String invalidSampleEP = "<inboundEndpoint xmlns=\"http://ws.apache.org/ns/synapse\"\n"
+            + "                                sequence=\"TestIn\"\n" + "onError=\"fault\"\n"
+            + "                protocol=\"http\"\n" + "suspend=\"false\">\n" + "<parameters>\n"
+            + "    <parameter name=\"inbound.http.port\">8085</parameter>\n" + "</parameters>\n" + "</inboundEndpoint>";
+    private static final String parentElm = "<sequence xmlns=\"http://ws.apache.org/ns/synapse\" name=\"TestIn\">\n"
+            + "    <send receive=\"receiveSeq\">\n" + "<endpoint>\n"
+            + "            <address uri=\"http://localhost:9000/services/SimpleStockQuoteService\"/>\n"
+            + "        </endpoint>\n" + "    </send>\n" + "</sequence>\n"
+            + "<sequence xmlns=\"http://ws.apache.org/ns/synapse\" name=\"receiveSeq\">\n" + "    <send/>\n"
+            + "</sequence>";
+
+    public void testCreateValidInboundEP() throws Exception {
+        ep = factory.createInboundEndpoint(AXIOMUtil.stringToOM(sampleEP), config);
+        Assert.assertNotNull("Inbound Endpoint is null", ep);
+        Assert.assertEquals(ep.getName().equals("HttpListenerEP1"), true);
```

Comment:
Shall we change these asserts like below? 
```java
        Assert.assertEquals("message",  "HttpListenerEP1", ep.getName());
```
Let's fix in other places as well.

Reply:
Fixed with f5af1ad